tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -270,8 +270,9 @@ public:
    virtual std::vector<RFieldValue> SplitValue(const RFieldValue &value) const;
    /// The number of bytes taken by a value of the appropriate type
    virtual size_t GetValueSize() const = 0;
-   /// For many types, the alignment requirement is equal to the size; otherwise override.
-   virtual size_t GetAlignment() const { return GetValueSize(); }
+   /// As a rule of thumb, the alignment is equal to the size of the type. There are, however, various exceptions
+   /// to this rule depending on OS and CPU architecture. So enforce the alignment to be explicitly spelled out.
```

Comment:
It seems to be declared but not use yet, is that the intent?

Reply:
Not sure I understand.  Do you mean `GetAlignment()`? This is overwritten in the descendant classes.